twitter_scrooge/twitter_scrooge.bzl

```diff
@@ -314,6 +318,31 @@ scrooge_scala_library = rule(
     provides = [DefaultInfo, ScroogeInfo, JavaInfo],
 )
 
+def _create_scala_struct(ctx):
+  """Create a scala provider in the shape expected by the IntelliJ bazel plugin."""
+  output_jars = []
+
+  if ctx.attr.exports:
```

Comment:
I don't see why we want the exports here. The exports should be another target, so can't the intellij plugin see those? I don't even think we are using exports in scrooge anywhere internally.

Can we remove this part?

Reply:
Yup I think you're right. Removed.